eth/handler.go

```diff
@@ -473,6 +461,28 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		}
 		return p.SendBlockBodiesRLP(bodies)
 
+	case p.version >= eth62 && msg.Code == BlockBodiesMsg:
+		// A batch of block bodies arrived to one of our previous requests
```

Comment:
OK, reason I was asking was just to improve the code comment - it reads as if the message will always be in response to node's request. Actually the comment can probably be removed, since the logic for ignoring unrequested messages is in the downloader.